kahuna/public/js/components/gr-panel/gr-panel.html

```diff
@@ -175,8 +175,8 @@
 
         <div class="image-info__group">
             <dl class="image-info__group--dl metadata-line">
-                <dt class="image-info__wrap metadata-line image-info__byline metadata-line__key image-info__group--dl__key--panel">by</dt>
-                <dd class="image-info__wrap metadata-line image-info__byline metadata-line__info image-info__group--dl__value--panel">
```

Comment:
I think this should be displayed if missing and user can edit (i.e. provide the ability to add one if it's not there and you are allowed to).